ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -1044,6 +1044,33 @@ then
 	fi
 
 	#
+	# RC car setup
+	#
+	if [ $VEHICLE_TYPE == rccar ]
```

Comment:
In current PX4 we have FW, MC, and VTOL. Could we call this rover (RV) or something else?